applications/tokenomics-survey-2022.md

```diff
@@ -0,0 +1,327 @@
+| **0e.** | Article | We will send a **draft article** (with source code), in the English language.|
+to test all the functionality delivered with this milestone. |
+| **0e.1**    | Article                   | We will publish an **working paper** as indicated above. |
+| **0e.2**   |  - Section: Published models    | Initial annotated bibliography |
+| **0e.3**   |  - Section: Polkadot Parachain Economies | Parachain summary updated with references to published models |
+
+### Milestone 3 — Finalize Article
+
+- **Estimated Duration:** 1 month
+- **FTE:**  0.2
+- **Costs:** 4,000 USD
+
+Finalize the flow-chart/decision-tree that will help developers place their token in the following contexts:
+
+- Token Type
+- Model Type
+- Economy Type
+- Sector Type
+- Production Type
+- Monetary Type
+
+Finalize the annotated bibliography.
+Promote the working paper, incorporate feedback.
+The report/working paper will be posted to [SSRN (e.g. FEN - Cryptocurrency Research eJournal)](https://papers.ssrn.com/sol3/JELJOUR_Results.cfm?form_name=journalBrowse&journal_id=3839004), [IDEAS](https://ideas.repec.org/), [Arxiv (q-fin.GN)](https://arxiv.org/list/q-fin.GN/recent).
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | Copyright and Licenses | Creative Commons Attribution 4.0 International License (article), Dual Apache 2 or MIT License (code) |
+| **0b.** | Documentation/Tutorial | We will update both **artifacts documentation** of the deliverables and a basic **tutorial** that explains how a user can (for example) execute the code included or can visualize data or use any artifacts included. |
+| **0c.** | Methodology | Update the detailed explanation of how the results were achieved and how to reproduce/verify the results. |
+| **0d.** | Infrastructure | We will update the list of all infrastructure requirements (text editors with proper versions, software packages, data packages, etc) that can be used to verify the deliveries with this milestone. LaTeX for article production and Docker files for software execution. |
+| **0e** | Article | We will update a **draft article** (with source code), in the English language.|
+to test all the functionality delivered with this milestone. |
+| **0e.1**   |  - Section: Introduction           | Place the topic in perspective and motivate non-specialist readers.  Text and tables where relevant/appropriate. |
+| **0e.2**   |  - Section: Published models    | Update annotated bibliography |
+| **0e.3**   |  - Section: Polkadot Parachains | Parachain summary updated with references to published models |
+| **0e.4**   |  - Appendix: Methodology  | The research methodology.  Text and tables where relevant/appropriate. |
+| **0e.5**   |  - Section: Summary                | Where are we and where to next? |
```

Comment:
We are gently asking our applicants/future grantees that add an acknowledgments section to the end of the article with the statement: "This work was supported by a research grant from the Web3 Foundation.". Could you please add it to your deliverables?